src/cmdstan/arguments/arg_pathfinder.hpp

```diff
@@ -22,12 +22,14 @@ class arg_pathfinder : public arg_lbfgs {
         "save_single_paths", "Output single-path pathfinder draws as CSV",
         false));
     _subarguments.push_back(new arg_single_bool(
-        "psis_resample", "If true, perform psis resampling on samples returned"
+        "psis_resample",
+        "If true, perform psis resampling on samples returned"
         " from individual pathfinders. If false, returns num_paths * num_draws"
         " samples",
         true));
     _subarguments.push_back(new arg_single_bool(
-        "calculate_lp", "If true, individual pathfinders do not calculate lp."
+        "calculate_lp",
+        "If true, individual pathfinders do not calculate lp."
```

Comment:
this seems counter-intuitive - true == do it,  false == don't do it?

Reply:
Yes! I mixed up the docs, lp values are calculated for true and not for false